logistic_regression.md

```diff
@@ -0,0 +1,168 @@
+# Weighted Aggregate Logistic Regression
```

Comment:
You should credit the authors (you kept the acknowledgments, but not authors).  Assuming they are OK with that, of course.